packages/validators/src/client/index.ts

```diff
@@ -19,7 +19,8 @@ export {
 } from "../server/external";
 
 export const CreateReviewFormSchema = z.object({
-  restaurantId: z.string().pipe(z.coerce.number().int().positive()),
+  restaurantId: z.string(),
```

Comment:
```suggestion
  restaurantId: z.string().min(50).max(50),
```
you should try to restrict the value as much as possible (in this example, the length must be exactly 50)

Reply:
done